bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -320,8 +327,18 @@ public void startRefresh(int refresh) {
 
     public void updateRequest() throws IOException {
         getURLInput(BASE_URL);
-        if (liveChannelsLinked() && "true".equals(rtEnabled) && !isConnected()) {
-            startLiveStream();
+        if (liveChannelsLinked() && "true".equals(rtEnabled)) {
+            if (lastWebSocketMessage != null && lastWebSocketMessage.plusMinutes(5).isBefore(LocalDateTime.now())) {
+                logger.info("Got no data for 5 minutes from tibber. Last data from tibber on {}. Reconnect WebSocket.",
+                        lastWebSocketMessage);
+                close();
+                startLiveStream();
+            } else if (isConnected()) {
+                logger.debug("Sending Ping Message");
+                session.getRemote().sendPing(ByteBuffer.wrap("OpenHab Ping".getBytes(StandardCharsets.UTF_8)));
```

Comment:
```suggestion
                session.getRemote().sendPing(ByteBuffer.wrap("openHAB Ping".getBytes(StandardCharsets.UTF_8)));
```

Waht would the usual frequency be for this ping to be send?

Reply:
There is no usual frequency; In my opinion everything less than 5 minutes is ok. After 5 Minutes some firewall implementations drop connections when no data flow is detected.
I chose the update interval to no create another scheduler.